apps/angular-thirty-seconds/src/app/angular-thirty-seconds/create-snippet/create-snippet.component.html

```diff
@@ -37,6 +42,11 @@ <h1>New Snippet
               <mat-option value="advanced">advanced</mat-option>
             </mat-select>
           </mat-form-field>
+          <div *ngIf="snippetForm.get('level').touched && snippetForm.get('level').invalid">
```

Comment:
How would it be possible to not fill out the level? Isn't it a select?

Reply:
Yes, this is select and it's impossible not to fill it in
I added this check for case if  one day someone decides to replace select to something else :)